### DISCUSSION TITLE
Realtime - Listening to row level changes that are not some value

### DISCUSSION THREAD

**Discussion by vin-ce** [supabase/supabase#731] [Questions] [answered]
Hello,
Was wondering if it was possible to listen to row changes that _aren't_ of a particular value?
So from the docs, something like this:

const mySubscription = supabase
  .from('countries:id   _!=_  eq.200')
  .on('UPDATE', handleRecordUpdated)
  .subscribe()
  
 I'm assuming filters are unavailable.
Thanks!

**Comment by kiwicopple** [comment, marked as answer]
Hey @vin-ce, unfortunately we don't offer this yet

> I'm assuming filters are unavailable.

Correct - our realtime server is a limited to basic filters (for now)

**Reply by vin-ce** [reply]
No worries, thanks for the response!
